repose-aggregator/artifacts/docker/src/docker/resources/file/centos/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Use CentOS as a base, to enable security patching for CentOS users.
+# The version should be whatever version conforms with the managed CentOS image.
+FROM centos:7
+
+MAINTAINER The Repose Team <reposecore@rackspace.com>
+
+# Setting constants
+ENV APP_ROOT=/etc/repose
+ENV APP_VARS=/var/repose
+ENV APP_LOGS=/var/log/repose
+ENV PKG_DIR=/tmp/packages/repose/
+
+# This environment variable is used to set command-line options.
+# The user can manually set these options using the "-e" flag with the Docker run command.
+ENV JAVA_OPTS=
+
+# Copy local Repose packages
+COPY repose-*.rpm ${PKG_DIR}
+COPY repose-filter-bundle-*.rpm ${PKG_DIR}
+COPY repose-extensions-filter-bundle-*.rpm ${PKG_DIR}
+COPY repose-experimental-filter-bundle-*.rpm ${PKG_DIR}
```

Comment:
Same here.

Reply:
And in the `repo` versions too.

Reply:
Never mind. That is way wrong. I was dealing with a locally modified version of the repository Docker files that is now similar to the local flavor.

Reply:
Nope, you are absolutely right. The `ARG` instruction would be given a different value which would force all subsequent instructions, including the installation of the JDK, to run. I'll go ahead and move that down as well, for the sake of efficiency.